tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -166,6 +176,37 @@ RHODS Dahsboard Pod Should Contain OauthProxy Container
         List Should Contain Value    ${container_name}    oauth-proxy
     END
 
+Is Environment Staging
+    [Documentation]    Return true if environment is staging otherwise fails
+    ${list_of_links} =    Get Links From Switcher
+    Should Contain    ${list_of_links}[0]   qaprodauth
+
+Check Application Switcher Links To Openshift Cluster Manager
+    [Arguments]    ${link}    ${cluster_id}
+    ${status}    Run Keyword And Return Status    Is Environment Staging
+    ${cluster_id} =    Remove String    ${cluster_id}    "
+     Should Be Equal    ${link}    https://cloud.redhat.com/openshift/details/${cluster_id}
+    IF    "${status}" == "True"
+        Check HTTP Status Code    https://qaprodauth.cloud.redhat.com/openshift/details/${cluster_id}
+    END
+
+Get Links From Switcher
+    [Documentation]    Returns the OpenShift Console and OpenShift Cluster Manager Link
+    ${list_of_links} =    Create List
+    ${link_elements}=    Get WebElements    //a[@class="pf-m-external pf-c-app-launcher__menu-item" and not(starts-with(@href, '#'))]
+    FOR    ${ext_link}    IN    @{link_elements}
+        ${href}=    Get Element Attribute    ${ext_link}    href
+        Append To List    ${list_of_links}    ${href}
+    END
+    [Return]    ${list_of_links}
+
+Check Application Switcher Links To Openshift Console
+    [Documentation]    Checks the HTTP status of OpenShift Console
+    [Arguments]    ${link}
+    ${status} =    Check HTTP Status Code    ${link}
+    Should Be Equal    ${status}    ${200}
```

Comment:
I think this should also check that the link correspond to the expected one (which can be taken from the test-variables or fetched by using oc command)